config/locales/en/documents/history.yml

```diff
@@ -16,7 +16,7 @@ en:
         image_removed: "Removed image"
         new_edition: "New edition"
         create_preview: "Create preview"
-        retired: "Retired"
+        withdrawn: "Retired"
```

Comment:
Did we decide at a product level not to change these? I just wonder if it's simpler to do them all particularly as bits of the naming creep out in URLs.

Reply:
No, I just thought maybe the user facing change should happen at the same time as the functionality itself.  Happy to change now though.

Reply:
Yeah, should be fine to go whole way with it.

Reply:
Now updated 👍